spec/integration/fork/after/gen/Foo/App-iOS/main.m

```diff
@@ -1,4 +1,2 @@
-@import Foundation;
```

Comment:
in our own `main.m` generation we no longer import those as cocoapods does as they are unneeded anyway.